### DISCUSSION TITLE
Include tabulator in anvil extras library

### DISCUSSION THREAD

**Discussion by steininge** [anvilistas/anvil-extras#228] [Ideas]
The tabulator component is really good. Would it make sense to include this in the anvil-extras library?
https://anvil.works/forum/t/tabulator-with-anvil-components/4646

**Comment by s-cork** [comment]
It might - though it needs a little bit of work to make it more friendly for anvil-extras.

For now I've added it to the third party apps and you can add it with the code `TGQCF3WT6FVL2EM2`
